README.md

```diff
@@ -239,6 +236,17 @@ It should look something like this for ABP:
 - Click Save
 - Turn on the device and you are ready to go
 
+### Optional device properties
+
+Customizations to lora devices are set by creating specific twin desired properties on the device. The following customizations are available:
+
+|Name|Description|Configuration|When to use|
+|-|-|-|-|
+|Enable/disable downstream messages|Allows disabling the downstream (cloud to device) for a device. By default downstream messages are enabled| Add twin desired property `"Downlink": false` to disable downstream messages. The absence of the twin property or setting value to `true` will enable downlink messages.|Disabling downlink on devices decreases message processing latency, since the network server will not look for cloud to device messages when an uplink is received. Only disable it in devices that are not expecting messages from cloud|
```

Comment:
acknowledgement of confirmed upstream are sent to devices even when downlink is set to false